distribution/client/test/storage.test.ts

```diff
@@ -56,11 +56,14 @@ describe('The storage module', () => {
       const pluginPath = Storage.pluginsDirectory();
       mkdirp.sync(pluginPath);
       filenames.forEach((filename) => {
+        fs.mkdirSync(`${pluginPath}/${filename}`)
```

Comment:
missing a `touchFile` in that directory to actually imitate real conditions. 

`fs.rmdir` calls http://man7.org/linux/man-pages/man2/rmdir.2.html which behaves like the `rmdir` command line, failing on non empty directories.

Reply:
I updated the test to touch a file in the directory and confirm it was all removed.

Reply:
I've tested all this locally, and I strongly suspect tests are broken/ineffective for some reason. I replaced all the `.resolves` below by `.rejects` below, and still got a green build. 

However, I might still be wrong and all this is working! :)
But then as discussed on the chat, I would please request at least an explanation on why my blabbering above is wrong (about `rmdir` in Node having the same behavior as the usual command-line `rmdir`).

Thanks!